version.go

```diff
@@ -1,12 +1,49 @@
 package ftl
 
-import "regexp"
+import (
+	"regexp"
+	"strconv"
+)
+
+const VersionNumberParts int = 3
 
 // IsRelease returns true if the version is a release version.
 func IsRelease(v string) bool {
 	return regexp.MustCompile(`^\d+\.\d+\.\d+$`).MatchString(v)
 }
 
+// IsVersionAtLeastMin returns true if any of the following are true:
+//   - minVersion is not defined (i.e. is emptystring)
+//   - v or minVersion is not a release version
+//   - v > minVersion when both v and minVersion are release versions
+func IsVersionAtLeastMin(v string, minVersion string) (bool, error) {
+	if minVersion == "" {
+		return true, nil
+	}
+	if !IsRelease(v) || !IsRelease(minVersion) {
+		return true, nil
+	}
+	vParsed := regexp.MustCompile(`\d+`).FindAllString(v, VersionNumberParts)
+	minVParsed := regexp.MustCompile(`\d+`).FindAllString(minVersion, VersionNumberParts)
+	for i := range VersionNumberParts {
```

Comment:
You'll need to explicitly check that the length of the output slice is `== VersionNumberParts`, otherwise this will panic if, for example, the version is `1.2`

Reply:
Ah yes, thanks for catching that. I removed this line in favor of using semver so now we can just rely on the IsRelease check above.